### PR TITLE
Add missing types of test-suite nodes.

### PR DESCRIPTION
**Pull request by gkalnytskyi**
I noticed that official documentation for the NUnit 3 XML results file is missing two types for `test-suite` node.
One example was provided in XRay documentation: https://docs.getxray.app/display/XRAY/Taking+advantage+of+NUnit+XML+reports , while another one I spotted when I was looking through Specflow/NUnit test results file.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/nunit/docs?pullRequest=520) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/nunit/docs?pullRequest=520) before we can accept your contribution.<br/><br/>:x: gkalnytskyi [sign now](https://cla.dotnetfoundation.org/nunit/docs?pullRequest=520)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/nunit/docs?pullRequest=520) it.</sub>

**Comment by SeanKilleen** [comment]
@ChrisMaddock this looks correct to me but I'm unsure. Can you confirm?

**Comment by ChrisMaddock** [comment]
Hi Sean, 

This is framework code, so not really my area. I’d check the code, but it’s a little tricky on my phone.  @CharliePoole - I know you’ve worked with these xml attributes for the GUI recently, do you happen to remember what’s what? 😊

If not, I’ll have a look into the code next time I’m at a computer. 😊

**Comment by CharliePoole** [comment]
As far as I know, the following elements are possible:

``` 
Project, Assembly, TestSuite, TestFixture, GenericFixture, ParameterizedFixture, SetUpFixture, GenericMethod, ParameterizedMethod, Theory
```

The engine creates `test-suite` elements of type `Project`. All the rest come from the framework and are defined in `TestDefinitions.xsd`. It may be that the schemas should be maintained in a separate repo, since at least some of them relate to the engine rather than or in addition to the framework.

File: docs/articles/nunit/technical-notes/usage/Test-Result-XML-Format.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ Represents a filter that examines the test id, which is generated by NUnit.
 * **Containing Elements:** [`<test-run>`](#test-run), [`<test-suite>`](#test-suite)
 * **Contained Elements:** [`<environment>`](#environment), [`<settings>`](#settings), [`<properties>`](#properties), [`<reason>`](#reason), [`<failure>`](#failure), [`<output>`](#output), [`<attachments>`](#attachments), [`<test-suite>`](#test-suite), [`<test-case>`](#test-case)
 * **Attributes**
-  * **type** The type of suite represented by this element. Currently supported types are Assembly, TestSuite, TestFixture,
+  * **type** The type of suite represented by this element. Currently supported types are Assembly, TestSuite, TestFixture, SetUpFixture, ParameterizedMethod.
```
Comment (SeanKilleen) [inline docs/articles/nunit/technical-notes/usage/Test-Result-XML-Format.md:117]:
```suggestion
  * **type** The type of suite represented by this element. Currently supported types are Project, Assembly, TestSuite, TestFixture, GenericFixture, ParameterizedFixture, SetUpFixture, GenericMethod, ParameterizedMethod, Theory.
```

Updating based on @CharliePoole's input.

**Comment by SeanKilleen** [comment]
Thanks @gkalnytskyi for the PR! Made a modification based on Charlie's input, and merging it now. 👍

**Comment by gkalnytskyi** [comment]
Thank you!

**Pull request merged**